orttraining/tools/amdgpu/Dockerfile.rocm4.1.pytorch

```diff
@@ -73,7 +73,8 @@ RUN git clone https://github.com/openucx/ucx.git \
   && cd build \
   && ../contrib/configure-opt --prefix=$UCX_DIR --without-rocm --without-knem --without-cuda \
   && make -j"$(nproc)" \
-  && make install
+  && make install \
+  && make clean
 
```

Comment:
Why don't delete the whole directory instead?

Reply:
Done, updated to remove the whole build directories instead.